web/tendrl/multiuser_cluster_import.rst

```diff
@@ -1,2 +1,52 @@
-multiuser_cluster_import
-*******************************************************************************
\ No newline at end of file
+Multiuser - Cluster import
+****************************
+
+:authors:
+          - mkudlej@redhat.com
+
+
+Description
+============
+
+Import cluster simultaneously from two browsers.
```

Comment:
This looks like the rare example of a test case, when a more deliberate explanation of the intent in the description section would be welcome.